src/TBuffer.cpp

```diff
@@ -2611,110 +2328,245 @@ void TBuffer::logRemainingOutput()
     mpHost->mpConsole->mLogStream.flush();
 }
 
-// returns how many new lines have been inserted by the wrapping action
-int TBuffer::wrapLine(int startLine, int screenWidth, int indentSize, TChar& format)
+inline int TBuffer::binarySearchHorizontalAdvance(const int& lineIndex, const int& indentSize, const QString& lineIndent, const int& screenWidth, const int& subStringStart, const int& lineCharTotal, const QFontMetrics& qfm, int& lineCharIterator, const bool isBefore)
```

Comment:
If this function/method has a return type of `int` then it should `return` something, as it doesn't this definition (and the declaration in the `TBuffer.h` file) had better be changed to:
```suggestion
inline void TBuffer::binarySearchHorizontalAdvance(const int& lineIndex, const int& indentSize, const QString& lineIndent, const int& screenWidth, const int& subStringStart, const int& lineCharTotal, const QFontMetrics& qfm, int& lineCharIterator, const bool isBefore)
```

Reply:
good catch!  Ha I meant to return an int but changed my mind.